force-app/main/default/classes/NextStepFuture.cls

```diff
@@ -71,6 +116,10 @@ public with sharing class NextStepFuture {
 
         role.Name = 'CRO';
 
-        upsert role;
+        // Tried to bypass ApexCRUDViolation, but it doesn't help. I don't know why.
+        Schema.DescribeSObjectResult sor = UserRole.SObjectType.getDescribe();
+        if (sor.isCreateable() && sor.isUpdateable()) {
+            Database.upsert(role);
+        }
```

Comment:
- If you tried inserting with AccessLevel.User_mode and still got the warning about ApexCRUDViolation, that may just be a PMD issue but the code would be fine.

Reply:
Thanks, Tony! With AccessLevel.User_mode warning was gone!